zeebe/dynamic-config/src/test/java/io/camunda/zeebe/dynamic/config/api/ClusterConfigurationManagementApiTest.java

```diff
@@ -163,6 +164,24 @@ void shouldRemoveMembers() {
     assertThat(changeStatus.plannedChanges()).containsExactlyElementsOf(expected);
   }
 
+  @Test
+  void shouldPurgeCluster() {
+    // given
+    recordingCoordinator.setCurrentTopology(
+        initialTopology
+            .addMember(id1, MemberState.initializeAsActive(Map.of()))
+            .addMember(id2, MemberState.initializeAsActive(Map.of())));
+    final var request = new PurgeRequest(false);
+
+    // when
+    final var changeStatus = clientApi.purge(request).join().get();
+
+    // then
+    // TODO Why are the planned changes empty? Because the RecordingChangeCoordinator is used?
```

Comment:
@npepinpe Do you have an idea here? I see planned changes when starting the StandaloneBroker and curling the API, but I do not see them here. Only difference seems to be the coordinator implementation.

Reply:
I'm assuming this is why we're not asserting the changes? That was one of my comment, but I guess this is why :smile: 

I don't see why, I mean other tests have `plannedChanges` and they're not empty.

Reply:
Alright, found it - there is no partitions in the current topology, so nothing to leave/restore.

When asserting the changes, make sure to use hard coded values (like the other tests), so we can detect granular changes in the future.